File: guides/micronaut-microservices-services-discover-eureka/bookinventory/java/src/main/java/example/micronaut/Application.java
package example.micronaut;

import io.micronaut.runtime.Micronaut;

import static io.micronaut.context.env.Environment.DEVELOPMENT;

public class Application {

    public static void main(String[] args) {
        Micronaut.build(args)
                .mainClass(Application.class)
                .defaultEnvironments(DEVELOPMENT)
                .start();
    }
}

File: guides/micronaut-oracle-cloud-streaming/chess-game/kotlin/src/main/kotlin/example/micronaut/Application.kt
package example.micronaut

import io.micronaut.runtime.Micronaut.build
import io.micronaut.context.env.Environment.DEVELOPMENT

fun main(args: Array<String>) {
	build()
	    .args(*args)
		.packages("example.micronaut")
		.defaultEnvironments(DEVELOPMENT)
		.start()
}
